src/open_inwoner/userfeed/summarize.py

```diff
@@ -0,0 +1,61 @@
+from typing import Iterable, NamedTuple, Optional
+
+from django.utils.translation import ngettext
+
+from open_inwoner.userfeed.adapter import FeedItem
+from open_inwoner.userfeed.choices import FeedItemType
+
+
+class SummaryLine(NamedTuple):
+    text: str
+    count: int
+
+
+def summarize_case_status_changed(
+    items: Iterable[FeedItem],
+) -> Optional[SummaryLine]:
+    num_items = sum(
+        1 for item in items if item.type == FeedItemType.case_status_changed
+    )
+    if num_items:
+        return SummaryLine(
+            ngettext(
+                "In {count} case the status has changed",
+                "In {count} cases the status has changed",
+                num_items,
+            ),
+            num_items,
+        )
+
+
+def summarize_action_required(items: Iterable[FeedItem]) -> Optional[SummaryLine]:
+    num_items = sum(1 for item in items if item.action_required)
```

Comment:
`num_items = len([item for item in items if item.action_required])` is faster than performing `sum`. It requires space for the list, but that shouldn't be an issue.

Reply:
I don't think speed is an argument if you're fine allocating throwaway lists.

 This is more descriptive so I'm keeping it.